Dockerfile

```diff
@@ -13,6 +13,7 @@ COPY package*.json .
 RUN npm ci --omit=dev
 
 COPY src ./src
```

Comment:
Renaming `src` to `frontend` seems pretty good to me, the nit with that is the number of changes it will cause, like changing the relative path in all files... 😅.